src/components/document/document-workspace.tsx

```diff
@@ -188,6 +190,23 @@ export class DocumentWorkspaceComponent extends BaseComponent<IProps, {}> {
     }
   }
 
+  private handleCopyDocument = (document: DocumentModelType) => {
+    this.stores.ui.prompt(`Give your workspace copy a new name:`,
```

Comment:
```suggestion
    this.stores.ui.prompt(`Give your document copy a new name:`,
```
We should get final wording from Leslie or Michael, but I suspect "document" will make more sense to users than "workspace".

Reply:
"Workspace" actually came directly from the design specs (Student: Workspace New Copy Pop-ups)! Should I ask for feedback from them again?

Reply:
Nope. If that's from the specs then I stand corrected. Go with the specs. (I happen to think it's a poor choice of words, but that's a separate issue.)